### PR TITLE
Add Triplelift adapter to Prebid for US

### PR DESCRIPTION
**Pull request by ioanna0**
## What does this change?

Adds Triplelift Adapter to prebid header bidding only in US
Includes switch default state ON
Includes AB test on 0% 

## Screenshots

Screenshot 2019-08-05 at 17 39 23

### Tested

- [x] Locally
- [x] On CODE (optional)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "ophan-tracker-js": "1.3.13",
     "preact": "^8.2.9",
     "preact-compat": "^3.18.4",
-    "prebid.js": "https://github.com/guardian/Prebid.js.git#71d18d2",
+    "prebid.js": "https://github.com/guardian/Prebid.js.git",
```
Comment (janua) [inline package.json]: This will implicitly change when master updates, which is why we use direct commit references. So you might want to use the commit new/old ID.

File: static/src/javascripts/projects/commercial/modules/prebid/bid-config.js [hunk 2]
```diff
@@ -323,6 +327,26 @@ const getPangaeaPlacementId = (sizes: PrebidSize[]): number => {
     return 13892409; // Other Section MPU as fallback
 };
 
+const getTripleLiftInventoryCode = (
+    slotId: string,
+    sizes: PrebidSize[]
+): string => {
+    if (containsLeaderboard(sizes))
```
Comment (janua) [inline static/src/javascripts/projects/commercial/modules/prebid/bid-config.js]: Will the linter let you away with an `if` statement without curly brackets!? I always thought we kept them to be consistent, but was sure it was enforced by the linter!

File: package.json (hunk 1 above)
Comment (ioanna0) [reply package.json]: :+1: added it

File: static/src/javascripts/projects/commercial/modules/prebid/bid-config.js (hunk 2 above)
Comment (ioanna0) [reply static/src/javascripts/projects/commercial/modules/prebid/bid-config.js]: I noticed we are not consistent in using braces in 1 liner `if` statement. for example improve adapter uses  braces whereas Xaxis and TrustX don't use braces.

File: static/src/javascripts/projects/commercial/modules/prebid/bid-config.spec.js [hunk 3]
```diff
@@ -656,18 +657,6 @@ describe('triplelift adapter', () => {
...
-    test('should return correct triplelift adapter params for leaderboard', () => {
-        containsLeaderboard.mockReturnValueOnce(false);
-        containsMpu.mockReturnValueOnce(false);
-        containsDmpu.mockReturnValueOnce(true);
-        containsMobileSticky.mockReturnValueOnce(false);
-
-        const tripleLiftBids = bids('dfp-right', [[300, 600]])[1].params;
-        expect(tripleLiftBids).toEqual({
-            inventoryCode: 'theguardian_article_300x600_prebid',
-        });
```
Comment (janua) [inline static/src/javascripts/projects/commercial/modules/prebid/bid-config.spec.js]: Why was this removed in the end?

**Comment by janua** [comment]
LGTM :+1:

File: static/src/javascripts/projects/commercial/modules/prebid/bid-config.spec.js (hunk 3 above)
Comment (janua) [reply static/src/javascripts/projects/commercial/modules/prebid/bid-config.spec.js]: Oh strange! The diff it showed me was this being removed!

Comment (ioanna0) [reply static/src/javascripts/projects/commercial/modules/prebid/bid-config.spec.js]: yes i deleted it along with adding an extra condition on mpu slot. We only have an **article** inventoryCode for mbu so I have added isArticle condition on it. Tried to fix the test but the way it's set up it's very hard to mock it only for that particular test. The isArticle variable gets initialised the same time with the imports and even changing it later on it doesn't change. I will merge it for now and see if I can add this test back by re-structuring these tests on a subsequent PR
